docs/reference/indices/rollover-index.asciidoc

```diff
@@ -42,19 +42,39 @@ NOTE: To roll over an index, a condition must be met *when you call the API*.
 automatically roll over indices when a condition is met, you can use {es}'s
 <<index-lifecycle-management, index lifecycle management (ILM) policies>>.
 
-The API accepts a single alias name and a list of `conditions`. The alias must point to a write index for
-a Rollover request to be valid. There are two ways this can be achieved, and depending on the configuration, the
-alias metadata will be updated differently. The two scenarios are as follows:
-
- - The alias only points to a single index with `is_write_index` not configured (defaults to `null`).
-
-In this scenario, the original index will have their rollover alias will be added to the newly created index, and removed
-from the original (rolled-over) index.
-
- - The alias points to one or more indices with `is_write_index` set to `true` on the index to be rolled over (the write index).
-
-In this scenario, the write index will have its rollover alias' `is_write_index` set to `false`, while the newly created index
-will now have the rollover alias pointing to it as the write index with `is_write_index` as `true`.
+The rollover index API accepts a single alias name
+and a list of `conditions`.
+To request a rollover,
+the specified alias must point to a write index,
+indicated by the `is_write_index` value.
+Depending on this value,
+the rollover request updates the alias metadata as follows:
+
+. The alias points to a single index
+with an unconfigured `is_write_index` value,
+which defaults to `null`.
++
```

Comment:
This is still confusing, as it seemingly contradicts the prerequisite that the alias points to a write index. I *think* the point is that `is_write_index` only needs to be set (is only set) if the alias points to multiple indices. If it only points to one index, it is assumed to be the write index. I think this would be clearer if you made the distinction based on whether or not the alias is associated with more than one index. 

Maybe something like:

> If the specified alias is associated with a single index, the rollover operation creates a new index, adds the alias to the new index, and removes the alias from the original index. 
> 
> If the alias is associated with multiple indices, one of the indices must have `is_write_index` set to `true`. The rollover operation creates a new index, sets `is_write_index` to `true` on the new index, and sets `is_write_index` to `false` on the original index.

Reply:
Thanks @debadair. I adopted your suggestions with some minor changes with c6da9180c8b.